definitions/npm/execa_v0.10.x/flow_v0.75.x-/execa_v0.10.x.js

```diff
@@ -79,8 +79,8 @@ declare module 'execa' {
   }
 
   declare interface Execa {
-    [[call]](file: string, args?: $ReadOnlyArray<string>, options?: Options): ThenableChildProcess;
-    [[call]](file: string, options?: Options): ThenableChildProcess;
+    (file: string, args?: $ReadOnlyArray<string>, options?: Options): ThenableChildProcess;
+    (file: string, options?: Options): ThenableChildProcess;
```

Comment:
This is valid syntax, but something in our babel/eslint stack fails to parse it for some reason. Both syntaxes behave identically.